python/tests/integration/arcticdb/test_s3.py

```diff
@@ -24,3 +26,25 @@ def test_s3_storage_failures(mock_s3_store_with_error_simulation):
 
     with pytest.raises(StorageException, match="Unexpected error: S3Error#17"):
         lib.read(symbol_fail_read)
+
+
+# TODO: To make this test run alongside other tests we'll need to:
```

Comment:
Could something like this help? https://github.com/man-group/ArcticDB/pull/1409/files Could recreate it in a fixture and put it back to its normal state after the test finishes